README.md

```diff
@@ -36,8 +36,29 @@ Start contributing to SwiftSyntax see [this guide](CONTRIBUTING.md) for more inf
 ## Bazel
 
 SwiftSyntax provides an experimental [Bazel](https://bazel.build) build configuration, maintained by Keith Smiley. 
-To use it you can pull the source archive from the relevant release tag
-into your `WORKSPACE` and depend on the libraries you need from the
+To use it, you can pull the source archive from the relevant release tag
+into your `MODULE.bazel` file (preferred and recommended) with `bazel_dep`. Bzlmod support was added starting release of `509.0.0` and above. All available versions can be found in the [Bazel Central Registry](https://registry.bazel.build/modules/swift-syntax)
+
+```python3
+bazel_dep(
+    name = "swift-syntax",
+    version = "600.0.1",
+)
+```
+
+You can also pull source archive with `WORKSPACE` prior to `509.0.0` but note that it is preferred to use `MODULE.bazel` with latest version and also `WORKSPACE` is being deprecated as of Bazel 8 and unsupported on Bazel 9. To use `WORKSPACE` and swift-syntax, you can use `http_archive` as such
```

Comment:
this implies `509.0.0`+ can't be used with WORKSPACE with isn't true.

Reply:
yea thats true, i updated the comment w/o the version number so that isnt implied

Reply:
I also think we don't need to mention the WORKSPACE support removal. For any reason folks need to use WORKSPACE,  they will stay on Bazel 8 until those reasons are gone. Chances of those people learning about WORKSPACE removal in Bazel 9 from this doc are very low IMO.

Reply:
I think my thought process was to leave a note for future users to also encourage to bump to bzlmod but i see your point. We can update the docs again later if needed.  

removed the deprecated note 👍 thanks for the reviews!